Parsers/ASimWebSession/Parsers/vimWebSessionCiscoMeraki.yaml

```diff
@@ -0,0 +1,275 @@
+Parser:
+  Title: Web Session ASIM parser for Cisco Meraki
```

Comment:
Web Session ASIM Filtering parser for Cisco Meraki

Reply:
Done